src/Definition/Exasol.php

```diff
@@ -407,4 +408,32 @@ public function getBasetype(): string
         }
         return $basetype;
     }
+
+    public static function createByBasetype(string $basetype): self
```

Comment:
pridal bych to do interface

Reply:
Chtel jsem, ale ostatni backendy by potom museli vyhazovat `Not implemented.`...

Reply:
to by imho nemelo vadit, nikde jinde se to zatim pouzivat nebude

Reply:
pridano